integrationTests/testInitializer.go

```diff
@@ -301,16 +324,113 @@ func MakeDisplayTable(nodes []*TestProcessorNode) string {
 }
 
 // PrintShardAccount outputs on console a shard account data contained
-func PrintShardAccount(accnt *state.Account) {
-	str := fmt.Sprintf("Address: %s\n", hex.EncodeToString(accnt.AddressContainer().Bytes()))
+func PrintShardAccount(accnt *state.Account, tag string) {
+	str := fmt.Sprintf("%s Address: %s\n", tag, base64.StdEncoding.EncodeToString(accnt.AddressContainer().Bytes()))
 	str += fmt.Sprintf("  Nonce: %d\n", accnt.Nonce)
-	str += fmt.Sprintf("  Balance: %d\n", accnt.Balance)
+	str += fmt.Sprintf("  Balance: %d\n", accnt.Balance.Uint64())
 	str += fmt.Sprintf("  Code hash: %s\n", base64.StdEncoding.EncodeToString(accnt.CodeHash))
 	str += fmt.Sprintf("  Root hash: %s\n", base64.StdEncoding.EncodeToString(accnt.RootHash))
 
 	fmt.Println(str)
 }
 
+// CreateRandomAddress creates a random byte array with fixed size
+func CreateRandomAddress() state.AddressContainer {
+	buff := make([]byte, sha256.Sha256{}.Size())
+	_, _ = rand.Reader.Read(buff)
+
+	return state.NewAddress(buff)
+}
+
+// CreateRandomHexAddress returns a string encoded in hex with the given size
+func CreateRandomHexAddress(chars int) string {
+	if chars < 1 {
+		return ""
+	}
+
+	buff := make([]byte, chars/2)
+	_, _ = rand.Reader.Read(buff)
+
+	return hex.EncodeToString(buff)
+}
+
+// GenerateAddressJournalAccountAccountsDB returns an account, the accounts address, and the accounts database
+func GenerateAddressJournalAccountAccountsDB() (state.AddressContainer, state.AccountHandler, *state.AccountsDB) {
+	adr := CreateRandomAddress()
+	adb, _, _ := CreateAccountsDB(nil)
+	account, _ := state.NewAccount(adr, adb)
+
+	return adr, account, adb
+}
+
+// AdbEmulateBalanceTxSafeExecution emulates a tx execution by altering the accounts
+// balance and nonce, and printing any encountered error
+func AdbEmulateBalanceTxSafeExecution(acntSrc, acntDest *state.Account, accounts state.AccountsAdapter, value *big.Int) {
+
+	snapshot := accounts.JournalLen()
+	err := AdbEmulateBalanceTxExecution(acntSrc, acntDest, value)
+
+	if err != nil {
+		fmt.Printf("!!!! Error executing tx (value: %v), reverting...\n", value)
```

Comment:
can you remove the !!! from error message?

Reply:
Done.